### PR TITLE
Modify workflow

### PR DESCRIPTION
**Pull request by Irma12**

### CODE REVIEW

File: .github/workflows/weekly-check-versions.yml [hunk 1]
```diff
@@ -4,15 +4,39 @@ on:
...
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN  }}
+          commit-message: Update dependencies
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: update-dependecies
```
Comment (iain-b) [inline .github/workflows/weekly-check-versions.yml:28]: Can we have concurrent PRs open with this ? perhaps we should put the date and perhaps time in the branch name?

**Review by iain-b — APPROVED** [review]
Looks good to me, just a thought on branch names but it's a future improvement

File: .github/workflows/weekly-check-versions.yml (hunk 1 above)
Comment (Irma12) [reply .github/workflows/weekly-check-versions.yml:28]: I am not sure about concurrent PRs, what would be use case for those? Yeah, that's good idea, I will put the date in branch name :) @iain-b

Comment (iain-b) [reply .github/workflows/weekly-check-versions.yml:28]: I'm thinking we'll end up wanting to run this more regularly than once a week and perhaps we haven't merged a PR then the creation of the next PR fails as the branch already exists.

Comment (Irma12) [reply .github/workflows/weekly-check-versions.yml:28]: Good point, I think it will just update the existing branch and treat that as one PR.

**Pull request merged**